evap/student/views.py

```diff
@@ -340,7 +340,8 @@ def vote(request: HttpRequest, evaluation_id: int, dropout: bool = False) -> Htt
         for contribution, form_group in form_groups.items():
             for questionnaire_form in form_group:
                 questionnaire = questionnaire_form.questionnaire
-                for question in questionnaire.questions.all():
+                for assignment in questionnaire.questions.all().prefetch_related("question"):
```

Comment:
Ah, I see now, .questions gives the assignments. Hm, can we make it so .questions still gives the questions and .question_assignments gives the assignments? I would expect that most callers would just need .questions, with the benefit that they are correctly ordered, whereas only the questionnaire edit stuff should care about assignments